src/summarize.rs

```diff
@@ -69,8 +76,10 @@ impl SummarizationClient {
         // split message into lines and uniquefy lines
         let mut lines = message.lines().collect::<Vec<&str>>();
         lines.dedup();
-        let message = lines.join("\n");
+        let mut message = lines.join("\n");
 
+        message = self.commit_tanslate(&message).await?;
```

Comment:
```suggestion
        let message = lines.join("\n");

        let message = self.commit_tanslate(&message).await?;
```

You don't need the `mut` :)